fe/fe-core/src/main/java/com/starrocks/sql/ast/ShowStorageVolumesStmt.java

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+package com.starrocks.sql.ast;
+
+import com.google.cloud.hadoop.repackaged.gcs.com.google.common.base.Strings;
+import com.starrocks.catalog.Column;
+import com.starrocks.catalog.ScalarType;
+import com.starrocks.qe.ShowResultSetMetaData;
+import com.starrocks.sql.parser.NodePosition;
+
+public class ShowStorageVolumesStmt extends ShowStmt {
+    private final String pattern;
+    private static final String SV_COL = "Storage Volume";
```

Comment:
```suggestion
    private static final String SV_COL = "StorageVolumes";
```

Reply:
The name of column is defined by PR.

Reply:
Maybe we can rename this column later if necessary.